tests/e2e/specs/modules/analytics/setup-with-account-no-tag.test.js

```diff
@@ -29,13 +35,7 @@ const setReferenceUrl = async() => {
 describe( 'setting up the Analytics module with an existing account and no existing tag', () => {
 	beforeAll( async() => {
 		await page.setRequestInterception( true );
-		page.on( 'request', request => {
-			if ( ! request._allowInterception ) {
-
-				// prevent errors for requests that happen after interception is disabled.
-				return;
-			}
-
+		useRequestInterception( request => {
```

Comment:
How is this change related?

Reply:
We have been updating/enhancing older tests with newer utilities as we come back to them for consistency.